bcda/api/requests.go

```diff
@@ -42,10 +44,57 @@ func init() {
 	db.DB().SetMaxIdleConns(utils.GetEnvInt("BCDA_DB_MAX_IDLE_CONNS", 25))
 	db.DB().SetConnMaxLifetime(time.Duration(utils.GetEnvInt("BCDA_DB_CONN_MAX_LIFETIME_MIN", 5)) * time.Minute)
 	repository := postgres.NewRepository(db)
-	models.GetService(repository, cutoffDuration, utils.GetEnvInt("BCDA_SUPPRESSION_LOOKBACK_DAYS", 60))
+	svc = models.NewService(repository, cutoffDuration, utils.GetEnvInt("BCDA_SUPPRESSION_LOOKBACK_DAYS", 60))
 }
 
-func BulkRequest(resourceTypes []string, w http.ResponseWriter, r *http.Request, retrieveNewBeneHistData bool) {
+func BulkPatientRequest(w http.ResponseWriter, r *http.Request) {
+	resourceTypes, err := ValidateRequest(r)
+	if err != nil {
+		responseutils.WriteError(err, w, http.StatusBadRequest)
+		return
+	}
+	reqType := models.DefaultRequest // historical data for new beneficiaries will not be retrieved (this capability is only available with /Group)
+	bulkRequest(resourceTypes, w, r, reqType)
+}
+
+func BulkGroupRequest(w http.ResponseWriter, r *http.Request) {
```

Comment:
How will we validate when/if a client calls `/Group/runout` with `_since`?  Do we ignore `_since` or return an error?

Reply:
I think the runouts should leverage the _since parameter.
1. ACO makes initial request for runout data in 2020-02-15. No _since provided. Caller retrieves all claims data that completed by 2020-12-31.
2. ACO makes another request for runout data in 2020-03-15. _since = 2020-02-15. Caller retrieves all claims data that completed by 2020-12-31 but written after 2020-02-15.

Reply:
Ahh!  Gotcha.  Excellent.